server/core/templates/login_base.html

```diff
@@ -17,6 +17,11 @@
 			alt="Kanidm" class="kanidm_logo" />
 		(% endif %)
 		<h3>Kanidm</h3>
+		(% match display_ctx.reauth %)
+			(% when Some with (reauth) %)
+				<h5>Reauthenticating as (( reauth.username )) to access (( reauth.purpose ))</h5>
+			(% when None %)
+		(% endmatch %)
```

Comment:
This can just be `(% if let Some(reauth) = display_ctx.reauth %)` instead of a match

```suggestion
		(% match display_ctx.reauth %)
			(% when Some with (reauth) %)
				<h5>Reauthenticating as (( reauth.username )) to access (( reauth.purpose ))</h5>
			(% when None %)
		(% endmatch %)
```